crux-bench/src/crux/bench.clj

```diff
@@ -78,11 +103,28 @@
                  {:body (json/write-str {:text message})
                   :content-type :json})))
 
-(defn- result->slack-message [{:keys [time-taken-ms bench-type] :as bench-map}]
-  (format "*%s* (%s): `%s`"
-          (name bench-type)
-          (java.time.Duration/ofMillis time-taken-ms)
-          (pr-str (dissoc bench-map :bench-ns :bench-type :crux-node-type :crux-commit :crux-version :time-taken-ms))))
+(defn- result->slack-message [{:keys [time-taken-ms bench-type percentage-difference-since-last-run
+                                      minimum-time-taken-this-week maximum-time-taken-this-week
+                                      doc-count av-count bytes-indexed] :as bench-map}]
+  (cond-> (format "*%s* (%s, *%s%%*. 7D Min: %s, 7D Max: %s): `%s`"
```

Comment:
it's probably clear enough in this case, but I prefer not having functions like `str` (particularly, functions that take multiple parameters of the same type) within an arrow - makes the reader (well, me, at least) explicitly consider what the desired behaviour is

Reply:
we could make this clearer using a pattern like this
```clojure
(->> (concat [(format ...)]
             (when ingest?
               (map ...)))
     (string/join "\n"))
```
where the order is immediately obvious?